editor/src/clj/editor/pipeline/bob.clj

```diff
@@ -118,56 +114,11 @@
                                (progress/make-cancellable-indeterminate msg)
                                progress/done))))))))
 
-(defn- ->graph-resource-scanner [ws]
-  (let [res-map (->> (g/node-value ws :resource-map)
-                  (map (fn [[key val]] [(subs key 1) val]))
-                  (into {}))]
-    (reify IResourceScanner
-      (openInputStream ^InputStream [this path]
-        (when-let [r (get res-map path)]
-          (io/input-stream r)))
-      (exists [this path]
-        (if-let [r (get res-map path)]
-          (resource/exists? r)
-          false))
-      (isFile [this path]
-        (if-let [r (get res-map path)]
-          (= (resource/source-type r) :file)
-          false))
-      (scan [this pattern]
-        (let [res (->> res-map
-                    (map first)
-                    (filter #(PathUtil/wildcardMatch % pattern))
-                    set)]
-          res)))))
-
 (defn- project-title [project]
   (let [proj-settings (project/settings project)]
     (get proj-settings ["project" "title"] "Unnamed")))
 
-(defn- run-commands! [project evaluation-context ^Project bob-project commands render-progress! task-cancelled?]
-  (try
-    (let [result (ui/with-progress [render-progress! render-progress!]
-                   (.build bob-project (->progress render-progress! task-cancelled?) (into-array String commands)))
-          failed-tasks (filter (fn [^TaskResult r] (not (.isOk r))) result)]
-      (if (empty? failed-tasks)
-        nil
-        {:error {:causes (engine-build-errors/failed-tasks-error-causes project evaluation-context failed-tasks)}}))
-    (catch Exception e
-      {:exception e})))
-
-(defonce ^:private build-in-progress-atom
-  (add-watch
-    (atom false)
-    ::build-in-progress-watch
-    (fn build-in-progress-watch-fn [_ _ _ new-state]
-      (try
-        (set-verbose-logging! new-state)
-        (catch Exception _
-          ;; We can safely ignore any errors here since we've already
-          ;; thrown an exception in case we're unable to change it to
-          ;; false at the top of this file.
-          nil)))))
+(defonce ^:private build-in-progress-atom (atom false))
```

Comment:
I changed this to not force the verbose option on every bob invocation, but instead to do it only when the verbose option is set to true (which is `invoke!`'s default).